ee/localserver/zta.go

```diff
@@ -24,6 +29,13 @@ func (ls *localServer) requestZtaInfoHandlerFunc(w http.ResponseWriter, r *http.
 		return
 	}
 
+	// Validate origin
+	requestOrigin := r.Header.Get("Origin")
+	if _, ok := allowlistedZtaOriginsLookup[requestOrigin]; !ok {
+		w.WriteHeader(http.StatusForbidden)
```

Comment:
We should log something here

Reply:
Sure! I can sanitize+log the origin -- anything else?

Reply:
Added -- let me know if you'd like it updated!